modules/claims_api/spec/requests/v2/veterans/power_of_attorney_request_spec.rb

```diff
@@ -141,15 +145,15 @@
 
       describe 'auth header' do
         context 'when provided' do
-          it 'returns a 200' do
+          it 'returns a 202' do
```

Comment:
Do these `auth_header` specs and the other ones not in the ccg flow context make sense to keep? CCG is the only supported auth right now.

I was also curious how the multiple reps with the same poa code 500 error spec was passing given we aren't looking at duplicate poa codes in the controller. Locally the error getting thrown is related to a missing VCR cassette and doesn't seem related to the test representative setup. Seems like a false positive.

Reply:
Good catch! Carry over from old code.